lib/engine/game/g_18_mag/meta.rb

```diff
@@ -15,7 +15,7 @@ module Meta
         GAME_INFO_URL = 'https://github.com/tobymao/18xx/wiki/18Mag'
         GAME_LOCATION = 'Hungary'
         GAME_PUBLISHER = :lonny_games
-        GAME_RULES_URL = 'https://www.lonny.at/games/18magyarorsz%C3%A1g/'
+        GAME_RULES_URL = 'https://boardgamegeek.com/filepage/222188/18mag-rules-english'
```

Comment:
Is there a reason not to use the official rules from Lonny?
https://www.lonny.at/app/download/10197748784/18Mag_Rules_ENG_comp.pdf?t=1620544783